_config.yml

```diff
@@ -23,9 +23,12 @@ logoalt: 18F logo
 # Navigation
 # List links that should appear in the site sidebar here
 navigation:
-- text: Introduction
+- text: Information for vendors
   url: index.html
   internal: true
+- text: Information for buyers
+  url: buyers/
+  internal: true
 - text: How can I use the Agile BPA
```

Comment:
the use of "I" can be confusing b/c there are many audiences here: Vendor, Buyer, Project Owner, 18F admin, etc.